app/controllers/services/govbox_controller.rb

```diff
@@ -84,4 +90,23 @@ def ensure_live
       render template: 'errors/not_found', status: 404, layout: 'application'
     end
   end
+
+  def fetch_statutory_entries
+    response = RestClient::Resource.new("#{ENV.fetch('AUTOFORM_URL')}/api/corporate_bodies/search", timeout: 10).get(params: { q: "cin:#{params[:cin]}", private_access_token: ENV.fetch('AUTOFORM_PRIVATE_ACCESS_TOKEN') })
+    json = JSON.parse(response.body, symbolize_names: true).first
```

Comment:
normalne by som to tu takto returnol a vo views by som si vybral co treba

```suggestion
    JSON.parse(response.body, symbolize_names: true).first&.fetch('statutory')
```